.azurePipeline/cibuildwheel_steps.yml

```diff
@@ -1,11 +1,11 @@
 
 steps:
-- task: UsePythonVersion@0
 - bash: |
-    # Use this fork of cibuildwheel for macos c++11 support
-    # https://github.com/joerick/cibuildwheel/pull/156
-    # python -m pip install git+https://github.com/Czaki/cibuildwheel
-    python -m pip install cibuildwheel==0.12.0
+    python -m pip install -U wheel
+    # See https://github.com/joerick/cibuildwheel/pull/220
+    # for discussion on MacOS x64 builds
+    #python -m pip install cibuildwheel==1.1.0
+    python -m pip install git+https://github.com/joerick/cibuildwheel
```

Comment:
it's not a good idea to reference and unversioned resource in the CI. What if they happen to break it again?
Maybe we can just point to a known working commit, and then eventually change to a working release, once available.

Reply:
I'll confirm whether the latest release is actually okay, if not I'll pin to a commit.